test/hotspot/jtreg/compiler/vectorization/runner/ArrayIndexFillTest.java

```diff
@@ -35,6 +35,7 @@
  *                   -XX:+WhiteBoxAPI
  *                   compiler.vectorization.runner.ArrayIndexFillTest
  *
+ * @requires (os.simpleArch == "x64") | (os.simpleArch == "aarch64")
```

Comment:
Is this just for disabling IR tests on 32-bit x86?

Reply:
Hi @vnkozlov , @pfustc , For x86_32 I find that auto-vectorization support is inherited because corresponding scalar IR nodes are not being generated for 32 bit targets. Thus I have enabled some tests only for x86_64, will re enable them with 32 bit support.

Reply:
Agree.

Reply:
That's ok for now. But I'd like to mention that it has a side effect that correctness check would also be disabled on other architectures (such as riscv64).